Directory.Build.props

```diff
@@ -4,9 +4,16 @@
     <Authors>Microsoft.Toolkit</Authors>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <NoPackageAnalysis>true</NoPackageAnalysis>
-    <PackageIconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</PackageIconUrl>
-    <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/license.md</PackageLicenseUrl>
+  <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
+    <PropertyGroup>
+      <PackageLicenseFile>license.md</PackageLicenseFile>
+      <PackageIcon>nuget.png</PackageIcon>
+      <PackageIconUrl>nuget.png</PackageIconUrl>
+    </PropertyGroup>
+    <ItemGroup>
+     <None Include="license.md" Pack="true" PackagePath=""/>
+     <None Include="build\nuget.png" Pack="true" PackagePath="\"/>
```

Comment:
You put the image in a `images` folder in the one below, so we should be consistent and do the same here or also put this one in a folder.